src/main/java/com/revature/model/SkillList.java

```diff
@@ -0,0 +1,27 @@
+package com.revature.model;
```

Comment:
This is completely unnecessary. Should be removed.

Reply:
fixed

Reply:
The entire file is completely unnecessary I mean. It was used as a replacement for a List<Skill> for one of the Repositories, but is no longer needed at all. This class should be deleted.